packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/types/purchase_details.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'in_app_purchase_error.dart';
+import 'purchase_status.dart';
+import 'purchase_verification_data.dart';
+
+/// Represents the transaction details of a purchase.
+///
+/// This class unifies the BillingClient's [PurchaseWrapper] and StoreKit's [SKPaymentTransactionWrapper]. You can use the common attributes in
```

Comment:
remove the bits about unifying android and iOS

Reply:
done